src/BackupChain.cs

```diff
@@ -55,9 +64,9 @@ private BackupChain(IList<BackupMetadata> recentBackups)
     /// <summary>
     ///   Backups ordered to have a full restore chain.
     /// </summary>
-    public IEnumerable<BackupMetadata> OrderedBackups => _orderedBackups;
+    public IEnumerable<StripedBackupSet> OrderedBackups => _orderedBackups;
 
-    private static IEnumerable<BackupMetadata> MostRecentFullBackup(IEnumerable<BackupMetadata> backups)
+    private static StripedBackupSet MostRecentFullBackup(IEnumerable<StripedBackupSet> backups)

```

Comment:
Using the variable name `backups` is a bit misleading here, especially since we are using `stripedBackups` in the calling function.
Can you please change it to `stripedBackups` here and in the other functions for the similar data?